### PR TITLE
feat(enhanced-readabilities): option to disable animation

### LINKED ISSUE #233: feature request: option to disable animation for enhanced readabilities plugin
As described at https://discord.com/channels/1229292283657195520/1229309023556341822/1244631511068840039:

> Hi, is it possible to disable animations on Enhanced Readabilities?

We can add an extra UI related option for users to disable the transition animation for enhanced readabilities plugin during view transition.

### PR DESCRIPTION
**Pull request by Glomzzz**
feat(enhanced-readabilities): option to disable animation

Close #233

### CODE REVIEW

File: packages/vitepress-plugin-enhanced-readabilities/src/client/components/LayoutSwitch.vue [hunk 1]
```diff
@@ -101,13 +101,13 @@ watch(layoutMode, (val) => {
   if (!mounted.value)
     return
 
-  setClasses(val, true)
+  setClasses(val, true && !options.layoutSwitch?.disableAnimation)
```
Comment (nekomeowww) [inline packages/vitepress-plugin-enhanced-readabilities/src/client/components/LayoutSwitch.vue:104]: Why not just `!options.layoutSwitch?disableAnimation`?

```suggestion
  setClasses(val, !options.layoutSwitch?.disableAnimation)
```

File: packages/vitepress-plugin-enhanced-readabilities/src/client/components/LayoutSwitch.vue [hunk 2]
```diff
@@ -101,13 +101,13 @@ watch(layoutMode, (val) => {
...
   if (!supportedLayoutModes.includes(val))
     layoutMode.value = options.layoutSwitch?.defaultMode || LayoutMode.BothWidthAdjustable
 })
 
 watch(route, () => {
-  setClasses(layoutMode.value, true)
+  setClasses(layoutMode.value, true && !options.layoutSwitch?.disableAnimation)
```
Comment (nekomeowww) [inline packages/vitepress-plugin-enhanced-readabilities/src/client/components/LayoutSwitch.vue:110]: Why not just `!options.layoutSwitch?disableAnimation`?

```suggestion
  setClasses(layoutMode.value, !options.layoutSwitch?.disableAnimation)
```

**Comment by nekomeowww** [comment]
Rest of the code LGTM, thanks for contributing!

File: packages/vitepress-plugin-enhanced-readabilities/src/client/components/LayoutSwitch.vue (hunk 1 above)
Comment (Glomzzz) [reply packages/vitepress-plugin-enhanced-readabilities/src/client/components/LayoutSwitch.vue:104]: schoolboy code 🤣

**Pull request merged**